src/m3/functions.ts

```diff
@@ -20,6 +30,9 @@ export const isPlural = (multiplicity: Multiplicity): boolean =>
 
 type Typed = Link | Property
 
+/**
+ * @return The type of the given {@link Feature}
+ */
 export const type = (feature: Feature): FeaturesContainer | Datatype | typeof unresolved =>
```

Comment:
Does this indicate that we need some abstraction to represent these three elements?